src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1757,7 +1757,7 @@ NOTE: On Mac OS X, the default temporary folder is not shared between all users.
 To use a shared temporary folder, you need to set the _$TMPDIR_ environment variable and the Java property _java.io.tmpdir_
 to /tmp before starting the ProActive Node.
 
-NOTE: the same applies on Windows. To use a shared temporary folder, create a folder shared by all proactive users e.g. C:\TEMP and define
+NOTE: the same applies on Windows. To use a shared temporary folder, create a folder shared by allProActive users e.g. C:\TEMP and define
```

Comment:
there is a typo here

Reply:
damn, I must have deleted a space :( thanks for noticing